.gitignore

```diff
@@ -8,3 +8,4 @@
 *.swp
 /Selenium/
 /private_examples/
+examples/bugs/*
```

Comment:
Any particular reason not to use `private_examples` for this? I'd rather avoid introducing a custom excluded directory for each contributor.